covfee/client/types/node.ts

```diff
@@ -45,6 +45,7 @@ export interface NodeType extends AllPropsRequired<BaseNodeSpec> {
    * Status of the node
    */
   status: NodeStatus
+  progress?: number | null
```

Comment:
Is this an optional attribute, the `?` if so, is the one in `app_context` also optional? Or it always present but it is `none` sometimes?